src/bundle/Resources/public/scss/_links.scss

```diff
@@ -0,0 +1,16 @@
+.ez-link {
+    text-decoration: underline;
+    transition: all 0.2s $ez-admin-transition;
+    
+    &:hover {
+        color: $ibexa-color-primary;
+    }
+
+    &--white {
+        color: $ibexa-white;
+    }
+
+    &--medium-black {
```

Comment:
Bad, bad, `--medium-black` but you're using `color-font` inside, what if someone decides to change `color-font` to pink?

Reply:
```suggestion
    &--medium-dark {
```
or just `--dark`